docs/introduction/overview.mdx

```diff
@@ -45,7 +45,7 @@ Have you met anyone who loves developing in Airflow? That's why we designed an e
 - **Bring your favorite tools:** Write [dbt](/dbt/overview) models, use your favorite [IDE](/development/text-editor), [track changes with Git](/getting-started/setting-up-git), and much much more.
 
 <p align="center">
-  <img src= "media/overview/mage-build.gif"/>
+  <img src= "/media/overview/mage-build.gif"/>
```

Comment:
i remember you mentioned it works locally.
does it have different behaviors between local and production?

Reply:
Yes, I think so. We need to make sure they have that leading backslash